sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParserUtils.scala

```diff
@@ -68,6 +68,11 @@ object ParserUtils {
   /** Convert a string node into a string. */
   def string(node: TerminalNode): String = unescapeSQLString(node.getText)
 
+  /** Convert a string node into a string without unescaping. */
+  def stringWithoutUnescape(node: TerminalNode): String = {
+    node.getText.slice(1, node.getText.size - 1)
```

Comment:
The string rule in SqlBase.g4 forces that the input has always quotes at the starting and ending. I may add a comment here.